ee/modules/600-flant-integration/templates/madison/deployment.yaml

```diff
@@ -56,7 +56,7 @@ spec:
       containers:
       - name: nginx
         {{- include "helm_lib_module_container_security_context_read_only_root_filesystem" $ | nindent 8 }}
-        image: "{{ $.Values.global.modulesImages.registry }}:{{ $.Values.global.modulesImages.tags.flantIntegration.madisonProxy }}"
+        image: {{ include "helm_lib_module_image" (list $ "madisonProxy") }}
```

Comment:
Wrapping quotes let auto-format manifests simetimes :)

Reply:
if necessary, we can add quotes to the helm lib function in the future.

Reply:
This is not possible, because it will break this helper usage in cases like this:
```
"--acme-http01-solver-image={{ include "helm_lib_module_image" (list . "certManagerAcmeSolver") }}"
```

Reply:
Indeed, quoting will not solve the formatting problem. 
Anyway, it is not a blocker.